CONTRIBUTING.md

```diff
@@ -41,7 +41,7 @@ $ cargo test integration  # runs all integration tests
 $ cargo test e2e          # runs all e2e tests
 ```
 
-Similarly, to run cast tests make sure you are in `crates/cast` directory:
+Similarly, to run cast tests make sure you are in `crates/sncast` directory:
```

Comment:
```suggestion
Similarly, to run sncast tests make sure you are in `crates/sncast` directory:
```
Not sure, but I guess we should change all recurences? I'm afraid it might be a bit confusing for users if we use cast/sncast interchangeably. @Arcticae ?

Reply:
Ok this actually is recurring thing in this PR, so I won't add comment per every file that contains this discrepancy, as this does not have much sense. Let's decide on naming convention first, I will continue the review then.